src/cocoa/toga_cocoa/libs/appkit.py

```diff
@@ -221,6 +222,24 @@ class NSLayoutPriority(Enum):
 NSApplicationPresentationDisableProcessSwitching = 1 << 5
 NSApplicationPresentationDisableHideApplication = 1 << 8
 
+def NSColorUsingColorName(background_color):
+    return {
+        Red: NSColor.redColor(),
```

Comment:
If you use the most recent version of Rubicon and toga, these should all be properties, not method calls.